src/models/calendar-restful-model-collection.ts

```diff
@@ -105,4 +105,80 @@ export default class CalendarRestfulModelCollection extends RestfulModelCollecti
         return Promise.reject(err);
       });
   }
+
+  consecutiveAvailability(
+    options: {
+      emails: Array<string[]>;
+      duration: number;
+      interval: number;
+      start_time?: number;
+      startTime?: number;
+      end_time?: number;
+      endTime?: number;
+      buffer?: number;
+      free_busy?: Array<{
+        email: string;
+        object: string;
+        time_slots: Array<{
+          object: string;
+          status: string;
+          start_time: number;
+          end_time: number;
+        }>;
+      }>;
+      open_hours: Array<{
+        emails: string[];
+        days: number[];
+        timezone: string;
+        start: string;
+        end: string;
+        object_type: string;
+      }>;
+    },
+    callback?: (error: Error | null, data?: { [key: string]: any }) => void
+  ) {
+    const freeBusyEmails = options.free_busy
+      ? options.free_busy.map(freeBusy => freeBusy.email)
+      : [];
+    for (const openHour of options.open_hours) {
+      for (const email of openHour.emails) {
+        if (
+          !options.emails.some(row => row.includes(email)) &&
+          !freeBusyEmails.includes(email)
+        ) {
+          throw new Error(
+            'Open Hours cannot contain an email not present in the main email list or the free busy email list.'
+          );
+        }
+      }
+    }
+
+    return this.connection
+      .request({
+        method: 'POST',
+        path: `/calendars/availability/consecutive`,
+        body: {
+          emails: options.emails,
+          duration_minutes: options.duration,
+          interval_minutes: options.interval,
```

Comment:
does it make sense to add other naming options?, e.g., 
```suggestion
          duration_minutes: options.duration | options.duration_minutes | options.durationMinutes,
          interval_minutes: options.interval | options.interval_minutes | options.intervalMinutes,
```

Reply:
Technically, yes, but this is the "old" way of handling camelCase vs. snake_case. Our new refactor (latest PR is #274) will be opting for camelCase always while searlizing/deserializing appropriately. For the sake of consistency of the current way the SDK does things I've opted to keep it the same for now.

Reply:
perfect - looks like you're gtg then 😄